src/backend/modules/transcription/transcribe_mlx.py

```diff
@@ -53,6 +59,6 @@ def writefile(input):
         json.dump(input, file, indent=2)
 
 if __name__ == "__main__":
-    test_path = "out/audiofile.wav"
+    test_path = "audiofile.wav"
```

Comment:
I doubt this will work. The audio file is in any case a .mp3 file, as long as the extract_audio is downloading as mp3?
If there is a need for another file extension like .wav or .ogg, it needs to be changed in extract audio as well, if it is possible.